new/PRS14025Balca.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14025Balca" xml:lang="en" type="pers">
```

Comment:
He already has ID PRS2435Balcasa

Reply:
Oops! I don't understand, why I did not find it in the search before! 
Anyways, I will transfer all the information to the existing ID PRS2435Balcasa and the new one (PRS14025Balca) is to be deleted or not approved. Correct?